packages/website/src/components/DocContainer.tsx

```diff
@@ -31,6 +34,12 @@ export function DocContainer({ name, kind, excerpt, summary, children }: DocCont
 			<SyntaxHighlighter language="typescript" style={vs} codeTagProps={{ style: { fontFamily: 'JetBrains Mono' } }}>
 				{excerpt}
 			</SyntaxHighlighter>
+			{typeParams && typeParams.length > 0 && (
+				<>
+					<h3>Type Parameters</h3>
+					<TypeParamTable data={typeParams} />
+				</>
+			)}
```

Comment:
```suggestion
			{typeParams?.length ? (
				<>
					<h3>Type Parameters</h3>
					<TypeParamTable data={typeParams} />
				</>
			) : null}
```

Bigger friend of doing it this way, also makes it easier to read in JSX